src/test/ui/suggestions/attribute-typos-stable.rs

```diff
@@ -0,0 +1,5 @@
+// rustc-env:RUSTC_BOOTSTRAP=
```

Comment:
Is this the right way to force a stable-like compiler in a test? I can't tell if this is a bug in my code or in how I'm making it use stable.